lib/elixir/lib/application.ex

```diff
@@ -334,6 +334,22 @@ defmodule Application do
   end
 
   @doc """
+  Returns a list with information about the applications which are currently running.
+  """
+  @spec started_applications :: [tuple]
+  def started_applications do
```

Comment:
#3603 specifies `:application.which_applications/0` but I'm happy to add in `/1` as well :)